api/src/main/java/com/okta/authn/sdk/resource/FactorProvider.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
```

Comment:
These copyrights should have the original date (as copying a file retains it's copyright)

Reply:
Original file (generated by mgmt sdk) does not have any copyright date. My rationale here is since this file is newly introduced in this Auth SDK, the copyright date is `2020-Present`.

Reply:
Grab the date of the first generated file, and use that.  The copyright starts when the file was created, not when copied.

For example the original source of this file dates to 2017

Reply:
(likely the same most of these other files)

Reply:
Ok, as discussed on Slack - will use `2017-Present` on these files.